system-test/cluster.ts

```diff
@@ -67,6 +67,7 @@ describe('Cluster', () => {
         time_created: Date.now(),
       },
     });
+    console.log(`Test Instance Id: ${instanceId}`);
     await operation.promise();
```

Comment:
Can we log the cluster we query in checkMetadata (e.g. method starting line 40)?

Reply:
Sure. We can log the cluster id.